packages/iobio-react-components/src/utils/stats_util.mts

```diff
@@ -23,7 +23,7 @@ import fs from 'node:fs';
 
```

Comment:
Potentially stats_util deserves its own folder to separate it from constants/types etc.